csharp/src/SeedLang.Ast/AstNode.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 The Aha001 Team.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace SeedLang.Ast {
+  // The base class of all AST nodes.
+  public abstract class AstNode {
+    // Create the string representation of the AST node.
+    public override string ToString() => AstStringBuilder.AstToString(this);
```

Comment:
Google C# style:

> Expression body syntax - Don’t use on method definitions. This will be reviewed when C# 7 is live, which uses this syntax heavily.

I think Google's decision is a bit outdated. We can make our own choice here. If we prefer the other way, please clarify this in the exception list in /CONTRUBUTING.md. Thanks.

Here and the rest.

Reply:
Done